Assets/Resources/JSON/ShopItems.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I put the spreadsheet in the drive here: https://docs.google.com/spreadsheets/d/1_i83WH2rab07VKbKB1bmq50SCdre23BN0vC1DqqvWQk/edit